cli/commands/build.go

```diff
@@ -18,15 +19,13 @@ var buildCmd = &cobra.Command{
 	Args:  cobra.ExactArgs(0),
 	Run: func(cmd *cobra.Command, args []string) {
 		app := "go"
-		arg0 := "build"
-		arg1 := "-o"
-		arg2 := "run"
+		arg := []string{"build", "-o", "metric_cli"}
```

Comment:
I changed the compiled binary to "metric_cli" from "run" because we will need to make a bash file called "run" in the future